src/app/static/src/tests/modelOptions/mutations.test.ts

```diff
@@ -140,6 +140,18 @@ describe("Model run options mutations", () => {
         expect(state.valid).toBe(false);
     });
 
+    it("sets valid to true if optionsFormMeta is empty and options exist when payload has data", () => {
```

Comment:
Should really check here that it populates the metadata in the form correctly too (puts the right values in the controls). 
Could also check the invalid case too, where some required options are provided and are populated from the options, but not all (could happen in case where new option has been added since the zip file was saved).